com.valvesoftware.Steam.Utility.MangoHud.yml

```
@@ -52,6 +53,20 @@ modules:
 
       - python3-mako.json
 
+      - name: libNVCtrl
+        subdir: src/libXNVCtrl
+        no-autogen: true
+        no-make-install: true
+        build-commands:
+          - cp -a libXNVCtrl.so* ${FLATPAK_DEST}/lib
+          - install -D *.h -t ${FLATPAK_DEST}/include/NVCtrl
+        sources: &libNVCtrl_sources
+          - type: archive
+            url: "https://download.nvidia.com/XFree86/nvidia-settings/nvidia-settings-440.64.tar.bz2"
+            sha256: ed2592a36a868698c14736949e0958ee3c87a42a646fec77a865ecec957f3f73
+          - type: patch
+            path: patches/libxnvctrl_so_0.patch
```

```suggestion
            path: patches/libxnvctrl_so_0.patch
        cleanup:
          - "/include"
```

Doing that worked for me.

---

Please add a top-level `cleanup` key instead (in yaml root, not in module object).